cmd/juju/model/config.go

```diff
@@ -6,42 +6,42 @@ import (
 	"bytes"
 	"fmt"
 	"io"
-	"os"
-	"path/filepath"
-	"regexp"
+	"io/ioutil"
 	"sort"
 	"strings"
 
 	"github.com/juju/cmd/v3"
 	"github.com/juju/errors"
 	"github.com/juju/gnuflag"
 	"github.com/juju/schema"
+	"github.com/juju/utils"
 	"gopkg.in/juju/environschema.v1"
+	"gopkg.in/yaml.v3"
 
 	"github.com/juju/juju/api/client/modelconfig"
 	jujucmd "github.com/juju/juju/cmd"
 	"github.com/juju/juju/cmd/juju/block"
 	"github.com/juju/juju/cmd/juju/common"
+	"github.com/juju/juju/cmd/juju/config"
 	"github.com/juju/juju/cmd/modelcmd"
 	"github.com/juju/juju/cmd/output"
-	"github.com/juju/juju/environs/config"
+	econfig "github.com/juju/juju/environs/config"
```

Comment:
Nit: I think it'd be preferable to us the slightly longer but slightly more meaningful `envconfig` or `environsconfig`.